### ISSUE TITLE
How did you generate sample data?

### ISSUE THREAD

**Issue by RanchMobile** [satishp962/hindi_handwritten_characters_recognition#1]
Did you use any library to generate the sample image data?

**Comment by satishp962** [comment]
Hello, Thank you for checking this out. Can you please specify which sample data are you talking about?

**Comment by RanchMobile** [comment]
Image data that you are using for Training.

**Comment by satishp962** [comment]
> Did you use any library to generate the sample image data?

No. Everything is done using python script. We scanned some papers with handwritten characters, cropped them and pre-processed it.

**Comment by satishp962** [comment]
For cropping we used a software.
